src/vision/src/HSVFilterNode.cpp

```diff
@@ -0,0 +1,177 @@
+/*
+ * Created By: Robyn Castro
+ * Created On: June 17, 2017
+ * Description: Filters out everything but green.
```

Comment:
More apt description would be like:
`Filters an image in the HSV colourspace to a binary image`